ui.apps/src/main/content/jcr_root/apps/hclecomm/config.publish/com.hcl.ecomm.core.services.impl.LoginServiceImpl.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OsgiConfig"
+    loginservice.domainName.string="localhost:8081/magento/rest"
```

Comment:
@DeepaliAgrwl : why do we need port 8081 in publish ?

Reply:
removed Rishabh